style.css

```diff
@@ -313,26 +373,332 @@ h4{
+    margin: 10% 0 0 4%;
+    font-size: 1.5em;
+  }
+
+  .info-card {
+    margin: 0 20% 0 5%;
+    font-size: 0.5em;
+    width: 80%;
+  }
+
+  .card1 p {
+    font-size: 0.7em;
+    margin: 0 10% 0 0;
+  }
+
+  .languages {
+    margin: 7% 30% 0 5%;
+    width: 60%;
+  }
+
+  .project-link {
+    margin: 7% 0 27% 5%;
+    font-size: 0.8em;
+  }
+
+  .thumbnail .item-2 {
+    position: absolute;
+    right: 2em;
+    width: 42%;
+  }
+
+  .card2 {
+    width: 50%;
+    margin-right: 50%;
+    margin-left: 2%;
+  }
+
+  .card2 p {
+    font-size: 0.7em;
+    margin: 0 10% 0 1%;
+  }
+
+  .card2 h2 {
+    margin: 10% 0 0 4%;
+    font-size: 1.5em;
+  }
+
+  .about-section {
+    display: flex;
+    justify-content: space-between;
+    font-size: 2.5em;
+  }
+
+  .about-myself {
+    margin-right: 50%;
+  }
+
+  .about-title {
+    margin: 22% 0.6em 0 10%;
+    font-size: 0.8em;
+  }
+
+  .bottom-menus {
+    position: absolute;
+    right: 2em;
+    width: 50%;
+  }
+
+  .about-myself p {
+    font-size: 1rem;
+    margin: 3% 15% 0 10%;
+  }
+
+  .connect2 p {
+    font-size: 1rem;
+    line-height: 1.5rem;
+    margin: 2% 5%;
+    color: #7f8cff;
+  }
+
+  .media-icon2 {
+    margin: 2% 0 0 5%;
+    width: 20%;
+  }
+
+  .resume-link {
+    margin: 5% 5% 15%;
+  }
+
+  .languages-des {
+    margin: 20% 2.5em 0 1.5em;
+    width: 90%;
+  }
+
+  .languages-box {
+    display: flex;
+    flex-direction: row;
+    margin: 0 5em 0 0;
+  }
+
+  .languages-box .javascript-box {
+    display: flex;
+    flex-direction: column;
+    margin: 1.5em 0.5em;
+    padding: 1% 10% 10% 1%;
+  }
+
+  .languages-box .javascript-box p {
+    font-size: 0.9em;
+    padding-left: 0;
+    margin-top: 10%;
+  }
+
+  .languages-box .html-box {
+    display: flex;
+    flex-direction: column;
+    margin: 1.5em 0.5em;
+    padding: 1% 10% 10% 1%;
+  }
+
+  .languages-box .html-box p {
+    font-size: 0.8em;
+    padding-left: 0;
+    margin-top: 20%;
+  }
+
+  .languages-box .css-box {
+    display: flex;
+    flex-direction: column;
+    margin: 1.5em 0.5em;
+    padding: 1% 10% 10% 1%;
+  }
+
+  .languages-box .css-box p {
+    font-size: 0.8em;
+    padding-left: 0;
+    margin-top: 20%;
+  }
+
+  .frameworks-menu {
+    margin: 0.6em 2.5em 0 1.5em;
+    width: 90%;
+  }
+
+  .skills-menu {
+    margin: 0.6em 2.5em 0 1.5em;
+    width: 90%;
+  }
+
+  .contact-section {
+    background-image: url(src/img/contact\ form\ background\ shapes.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position-x: center;
+  }
+
+  .contact-title h4 {
+    margin: 3em 0.6em 0 0.6em;
+    font-size: 1.5em;
+    color: #fff;
+  }
+
+  form {
+    grid-column: 3/7;
+  }
+
+  button {
+    grid-column: 4/6;
+    font-size: 1em;
+    margin: 5% 8% 5% 38%;
+  }
+}
+
+@media (min-width: 992px) {
+  .media-icon {
+    margin: 0.8em 20% 20% 15%;
+    width: 14%;
+  }
+
+  .info-card {
+    margin: 0 10% 0 5%;
+    font-size: 0.5em;
+    width: 50%;
+  }
```

Comment:
- [ ] Nice work making the changes :+1: However it would be nice to reduce the gap between the project information as requested by the previous reviewer to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-06-10 13-56-00](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/91142432/950206fd-12a2-44da-b392-ee87d5cbd215)
### What it is supposed to look like :point_down:
![244861606-7494c236-8793-4bf4-8056-1d95e4ce8945](https://github.com/JhonatanErosa/Project-2_Portfolio/assets/91142432/2d8113f6-24d4-42fc-95ac-40453e6089b2)